Source/ConversationV1/Models/ConversationModel.swift

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension Conversation {
+    
+    /// A Conversation model
+    public struct ConversationModel: JSONDecodable {
```

Comment:
Is this model used anywhere? The only references I see to it are within this file and in the Xcode project. Can it be removed?